Assets/HoloToolkit/Sharing/Scripts/SDK/SystemRole.cs

```diff
@@ -0,0 +1,18 @@
+/* ----------------------------------------------------------------------------
+ * This file was automatically generated by SWIG (http://www.swig.org).
+ * Version 3.0.2
+ *
+ * Do not make changes to this file unless you know what you are doing--modify
+ * the SWIG interface file instead.
+ * ----------------------------------------------------------------------------- */
+
+namespace HoloToolkit.Sharing {
+
+public enum SystemRole {
+  SessionDiscoveryServerRole = 0,
```

Comment:
Did the default change for the entry? I think the previous default was Primary Client?